lib/data/readme.md

```diff
@@ -53,3 +55,44 @@ Example:
   "replacementVersion": "4.0.0"
 }
 ```
+
+## Tell Renovate where to find changelogs (`changelog-urls.json`)
+
+The `changelog-urls.json` has all the changelog information.
+
+Renovate nearly always finds, and displays, the changelog for a package update automatically.
+
+To find the changelog, Renovate needs the:
+
+- Name of the package
```

Comment:
Reference again the field name `packageName`, that the URLs are grouped by datasource and an example to explain this.